src/main.rs

```diff
@@ -424,7 +424,7 @@ mod tests {
             aver_eq!(tb, guest.get_cpu_count(), 1);
             aver_eq!(tb, guest.get_initial_apicid(), 0);
             aver!(tb, guest.get_total_memory() > 496_000);
-            aver!(tb, guest.get_entropy() >= 1000);
+            aver!(tb, guest.get_entropy() >= 900);
```

Comment:
Well there are two ways to solve this, either we lower this value or we wait a bit more, right?
I mean we have a virtio-rng device supposed to seed the entropy, right?

Reply:
We're getting the lower number 'cause we're booting quicker. Which is a good thing! There is no meaning in the number (other that very low being very bad.) The original 1000 is just something that worked before.

Reply:
Sounds good!